docs/PR_GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request Guide
+
+## General development rules:
+
+- All the changes of the code base are implemented through pull requests (PR) on master or other branches
+- When all next release features are implemented in master and master is stable enough release branch is made from it
+- Only PR with bugfix should be done on release. If it’s necessary it’s permitted to cherry-pick such bugfix PR to master
+- When a release is published one of developers makes a PR with all new commits of the release to master
+- All the further commits to release branch should be cherry-picked to the master by commits author
+- When a PR to any branch is about to merge, rebase and merge option should be used to make the code source tree flatter.
+
+
+## Writing good PR:
+
+- PR should be small and reflect only one idea, a feature, a bugfix or a refactoring. In most cases a PR of 100 lines or less is considered as a PR of normal size and a PR of 1000 lines as big one
+- If a PR implements two different features, refactoring or bugfixes it should be to split into several PRs
+- A description field of every PR should contain a description, links to a ticket or/and links to article to confluence.
```

Comment:
either end every point with a dot, or don't end any

Reply:
I'm removing all the periods.